src/main/java/io/anserini/qa/passage/IdfPassageScorer.java

```diff
@@ -38,33 +41,42 @@
   private final DirectoryReader reader;
   private final MinMaxPriorityQueue<ScoredPassage> scoredPassageHeap;
   private final int topPassages;
+  private final List<String> stopWords;
 
   public IdfPassageScorer(String index, Integer k) throws IOException {
     this.util = new IndexUtils(index);
     this.directory = FSDirectory.open(new File(index).toPath());
     this.reader = DirectoryReader.open(directory);
     this.topPassages = k;
     scoredPassageHeap = MinMaxPriorityQueue.maximumSize(topPassages).create();
+    stopWords = IOUtils.readLines(new FileInputStream("src/main/resources/io/anserini/qa/english-stoplist.txt"));
```

Comment:
Use getResource here. See: http://stackoverflow.com/questions/2593154/get-a-resource-using-getresource

With getResource, when we distribute Anserini as a jar, if it's packaged correctly, the stopwords list will still be available.